public/css/main.css

```diff
@@ -34,3 +40,59 @@ body {
 /*** ADD EVENT FORM ***/
 
 /*** CALENDAR ***/
+
+.cal-wrap {
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  left: 0;
```

Comment:
is this doing anything?

Reply:
Yea, position:fixed has different start positions on some browsers

Reply:
Oh wait sorry I see your point, since it's width 100%. Yea fair

Reply:
So I don't understand are we going to get rid of it ?

Reply:
Nah just leave it, it doesn't matter and it's safer than not having it